### PR TITLE
Bugfix/tslint-issues

### PR DESCRIPTION
**Pull request by kuda1992**
Notes: This fixes all the linting issues which are causing the build too fail.

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { BadgeColour, ContactDetailsDataModel, FilterConfig, TCDocument } from 'projects/exui-common-lib/src';
-import { GoogleTagManagerService } from 'projects/exui-common-lib/src/lib/services/google-tag-manager/google-tag-manager.service';
+import { GoogleTagManagerService} from 'projects/exui-common-lib/src/lib/services/google-tag-manager/google-tag-manager.service';
```
Comment (ronaldmansveld) [inline src/app/app.component.ts:3]: I guess the space at the end here was removed by accident?

Comment (kuda1992) [reply src/app/app.component.ts:3]: Ahh yes good spot

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "0.2.5",
+  "version": "0.2.9",
```
Comment (ronaldmansveld) [inline package.json:3]: Latest release is 1.0.0, so this should be a pre-release version and > 1.0.0

File: projects/exui-common-lib/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "0.2.5",
+  "version": "0.2.9",
```
Comment (ronaldmansveld) [inline projects/exui-common-lib/package.json:3]: As above

**Review by ronaldmansveld — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (kuda1992) [reply package.json]: addressed

**Review by udaydenduluri33 — APPROVED** [review]
LGTMk

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "0.2.5",
+  "version": "1.0.1",
```
Comment (ronaldmansveld) [inline package.json:3]: Still needs to be a pre-release version, not an actual version. (Actual version isn't used until we do a proper release)

**Review by ronaldmansveld — APPROVED** [review]

**Pull request merged**